RELEASE-NOTES.txt

```diff
@@ -3,6 +3,7 @@
 20.5
 -----
 * [*] Use larger thumbnail previews for recommended themes during site creation [https://github.com/wordpress-mobile/WordPress-Android/pull/16848]
+* [***] Block Editor: List block: Adds support for V2 behind a feature flag [https://github.com/WordPress/gutenberg/pull/42702]
```

Comment:
Since this is being released under a feature flag that won't be enabled until mid September, let's add `internal`.

Reply:
@geriux Good idea, I'll update - do you think the priority level is correct?

Reply:
Yeah I think that's a proper one 👍